src/css/index.css

```diff
@@ -863,4 +880,84 @@ header {
 
 .tooltip p {
   margin: 0;
-}
\ No newline at end of file
+}
+
+
+/* ---------------------------------------------------------------------- *\
+                              #FOOTER
+\* ---------------------------------------------------------------------- */
+
+
+footer {
+  height: 374px;
+  background: var(--color-black-gray);
+  color: var(--color-white);
+  display: flex;
+  font-size: 18px;
+  line-height: 24px;
+}
+
+
+.footer-1 {
+  margin: 38px 316px 0 151px;
+  width: 262px;
+}
+
+
+.footer-1-title {
+  color: var(--color-yellow);
+  font-size: 36px;
+  margin: 0;
+  margin-left: 12px;
+}
+
+
+.subcribe {
+  display: flex;
+}
+
+
+.subcribe-email-form {
+  width: 211px;
+  height: 48px;
+  padding-left: 18px;
+  background: var(--color-black-gray);
+  border: 1px solid var(--color-border-gray);
+}
+
+
+.footer-2 {
+  margin-top: 69px;
+}
+
+
+.footer-2-follow-us {
+  margin-left: 235px;
+  width: 331px;
+}
+
+
+.footer-2-socials {
+  display: flex;
+  justify-content: space-between;
+}
+
+
+footer ul {
+  display: flex;
+}
+
+
+footer ul li {
+  display: flex;
+  flex-direction: column;
+  margin-right: 121px;
+}
+
+
+footer ul li a {
+  text-decoration: none;
```

Comment:
should add reset.css to reset default style of a tag, you can add more reset style for ul, li, ...